pyracf/common/downstream_fatal_error.py

```diff
@@ -0,0 +1,66 @@
+"""Exception to use when no data is returned by IRRSMO00."""
+from typing import Union
+
+
+class DownstreamFatalError(Exception):
+    """
+    Raised when no xml string is returned by IRRSMO00.
+    """
```

Comment:
Update docstrings.

Reply:
Changed to

`python3 
   """
    Raised IRRSMO00 returns with a SAF Return Code of 8,
    indicating that the request could not be processed.
    """
    `